spec/images_spec.rb

```diff
@@ -3,43 +3,39 @@
 require 'spec_helper'
 
 RSpec.describe Assembly::Images do
-  after do
-    # after each test, empty out the input and output test directories
-    remove_files(TEST_INPUT_DIR)
-    remove_files(TEST_OUTPUT_DIR)
-  end
+  before { cleanup }
 
   it 'does not run if no input folder is passed in' do
-    expect{ described_class.batch_generate_jp2('') }.to raise_error
+    expect{ described_class.batch_generate_jp2('') }.to raise_error(RuntimeError)
   end
 
   it 'does not run if a non-existent input folder is passed in' do
-    expect{ described_class.batch_generate_jp2('/junk/path') }.to raise_error
+    expect{ described_class.batch_generate_jp2('/junk/path') }.to raise_error(RuntimeError)
   end
 
   it 'runs and batch produces jp2s from input tiffs' do
-    ['test1', 'test2', 'test3'].each { |image| generate_test_image(File.join(TEST_INPUT_DIR, "#{image}.tif")) }
+    ['test1', 'test2', 'test3'].each { |image| generate_test_image(File.join(TEST_INPUT_DIR, "#{image}.tif"), profile: 'AdobeRGB1998') }
     described_class.batch_generate_jp2(TEST_INPUT_DIR, output: TEST_OUTPUT_DIR)
     expect(File.directory?(TEST_OUTPUT_DIR)).to be true
     ['test1', 'test2', 'test3'].each { |image| expect(File.join(TEST_OUTPUT_DIR, "#{image}.jp2")).to be_a_jp2 }
   end
 
   it 'runs and batch add color profile descriptions input tiffs with no color profile descriptions' do
```

Comment:
this description makes no sense to me. Is it meant to be "runs in batch and adds color profile descriptions to input tiffs ..."   For this and descriptions below?

Reply:
tried to make descriptions clearer, moved tests into describe blocks based on the methods they are etesting